systers_portal/templates/meetup/base.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+
+{% block title %}
+  {{ object.name }}
+{% endblock %}
+
+{% block content %}
+  <div class="mt40"></div>
+  <div class="row">
+    <div class="col-md-12">
+      <h1>{{ object.name }}</h1>
+      <hr/>
+    </div>
+    <div class="col-md-9">
+      <ul class="nav nav-pills">
+        <li> <a href="#">Upcoming</a> </li>
+        <li> <a href="#">Past</a></li>
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown">Members <span class="caret"></span></a>
+          <ul class="dropdown-menu" role="menu">
+            <li role="presentation">
+              <a role="menuitem" tabindex="-1" href="#">All Members</a>
```

Comment:
What does "All Members" mean? Does that mean both members and organizers?
Now at the second glace this dropdown has a bit confusing naming, since we are trying to say "Here's a page with everybody, but one link goes to only members, another one to only organizers".

What do you think about leaving "Members" pill/tab as is. And then making the "Members" page to contain at the top the organizers (let's assume we will have max 10 organizers) and below them all the members. So?